cmd/bmx/write.go

```diff
@@ -42,7 +42,7 @@ func init() {
 	}
 
 	if userConfig.Profile == "" {
-		writeCmd.MarkFlagRequired("profile")
+		userConfig.Profile = "default"
 	}
```

Comment:
The `.Flags().StringVar(...)` call on line 33 gives the option for default values. You could do something like:
```go
writeCmd.Flags().StringVar(&writeOptions.Profile, "profile", "default", "aws profile name")
```

and delete this check altogether I believe. I don't have go on my dev machine, so I can't test.